packages/suite-base/src/util/formatKeyboardShortcut.test.ts

```diff
@@ -29,4 +29,9 @@ describe("formatKeyboardShortcut", () => {
     userAgent.mockReturnValue("Mac");
     expect(formatKeyboardShortcut("O", ["Shift", "Meta"])).toBe("⇧⌘O");
   });
+
+  it("formats shortcuts correctly Mac second test", () => {
```

Comment:
You can put more than 1 expect in the same test, as it is doing basically the same thing.

Reply:
You can add even more tests, with different letters and combinations

Reply:
Added all expects on the same Mac test and added more tests overall